ballerina/build.gradle

```diff
@@ -33,46 +51,114 @@ def stripBallerinaExtensionVersion(String extVersion) {
             return extVersion
         }
     } else {
-        return extVersion.replace("${project.ext.snapshotVersion}", '')
+        return extVersion.replace("${project.ext.snapshotVersion}", "")
+    }
+}
+
+apply plugin: 'io.ballerina.plugin'
+
+ballerina {
+    packageOrganization = packageOrg
+    module = packageName
+    langVersion = ballerinaLangVersion
+    testCoverageParam = "--code-coverage --coverage-format=xml --includes=*"
```

Comment:
For a connector build, we should pass the `buildOnDockerImage` parameter.

Reply:
Fixed with [acfafde7659710d8b170eca0f0509ba4a6cdc6fa](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/pull/112/commits/acfafde7659710d8b170eca0f0509ba4a6cdc6fa)